libraries/AP_HAL/AP_HAL_Boards.h

```diff
@@ -217,15 +217,15 @@
 #endif
 
 #ifndef HAL_MAX_CAN_PROTOCOL_DRIVERS
-#if defined(HAL_BUILD_AP_PERIPH) || defined(HAL_BOOTLOADER_BUILD)
+#if defined(HAL_BOOTLOADER_BUILD)
```

Comment:
This is a major change. I'm relabeling these in a way to allow multi protocol in AP_Periph.. but that define is used throughout the codebase incorrectly in many places.. Or at least that's my interpretation of it